GitUI/Interops/DwmApi/DwmApi.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Runtime.InteropServices;
+
+namespace GitUI.Interops.DwmApi
+{
+    /// <summary>
+    /// Allow to set dark title bar on Win10. From: https://stackoverflow.com/questions/57124243/winforms-dark-title-bar-on-windows-10/62811758#62811758
+    /// </summary>
+    internal static class DwmApi
+    {
+        [DllImport("dwmapi.dll")]
+        private static extern int DwmSetWindowAttribute(IntPtr hwnd, uint attr, ref int attrValue, int attrSize);
+
+        private const uint DWMWA_USE_IMMERSIVE_DARK_MODE_BEFORE_20H1 = 19;
+        private const uint DWMWA_USE_IMMERSIVE_DARK_MODE = 20;
+
+        internal static bool UseImmersiveDarkMode(IntPtr handle, bool enabled)
+        {
+            if (!IsWindows10OrGreater(17763))
```

Comment:
Consider either using a named argument:

```suggestion
            if (!IsWindows10OrGreater(build: 17763))
```

Or renaming to `IsWindows10BuildOrGreater(17763)`.

Ditto below.